abel/tools/polar.py

```diff
@@ -56,19 +56,19 @@ def reproject_image_into_polar(data, origin=None, Jacobian=False,
 
     ny, nx = data.shape[:2]
     if origin is None:
-        origin = (nx//2 + nx % 2, ny//2 + ny % 2)   # % handles odd size image
+        origin = (nx//2, ny//2) 
 
     # Determine that the min and max r and theta coords will be...
     x, y = index_coords(data, origin=origin)  # (x,y) coordinates of each pixel
     r, theta = cart2polar(x, y)  # convert (x,y) -> (r,θ), note θ=0 is vertical
 
-    nr = np.round((r.max()-r.min())/dr)
+    nr = np.int(np.ceil((r.max()-r.min())/dr))
 
     if dt is None:
         nt = max(nx, ny)
     else:
-        # dt in degrees
-        nt = np.round((theta.max()-theta.min())/(np.pi*dt/180))
+        # dt in radians
+        nt = np.int(np.ceil((theta.max()-theta.min())/dt))
```

Comment:
I agree with the choice of radians. There is a little more work, because `linbasex` uses degrees. I will look at.